examples/pipelines/simple_pipeline/simple_pipeline_fondant.py

```diff
@@ -0,0 +1,42 @@
+"""Pipeline used to create a stable diffusion dataset from a set of given images."""
+import logging
+
+from config.general_config import KubeflowConfig
```

Comment:
There is also a `config.py` file that looks the same as the `general_config.py`

Reply:
I think it's better to keep the general configs in `config.py` at the `pipelines` directory level instead of replicating it inside each pipeline example since it contains cloud and cluster specific configs applicable to all pipelines. I'll make the changes

```
pipelines
--config.py
--pipeline_1
----components_config.py
--pipeline_2
----components_config.py
```